cache/backend/backend.go

```diff
@@ -88,3 +89,30 @@ func InitializeFileSystemBackend(c FileSystemConfig, debug bool) (cache.Backend,
 
 	return newFileSystem(c.CacheRoot), nil
 }
+
+// CloudStorageConfig is a structure to store Cloud Storage backend configuration
+type CloudStorageConfig struct {
+	Bucket     string
+	ACL        string
```

Comment:
I don't think ACL is used in anywhere, let's remove it.

Reply:
Or use it, if we have an option to pass it.

Reply:
https://godoc.org/cloud.google.com/go/storage#ACLHandle.Set

Reply:
Ohhh thanks ! i didn't find how to set ACL last time i tried